spec/game_spec.rb

```diff
@@ -33,31 +33,6 @@
     expect(next_player).to eq('O')
   end
 
-  it 'ends in a tie if all spots are taken and there is winning combo' do
```

Comment:
Lets fix the spacing for the tests in this file to follow proper [testing practices](https://thoughtbot.com/blog/four-phase-test) that you are using in the other spec files

Reply:
spacing has been resolved